Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,110 @@
+Code Review Etiquette
+===========
+
+This document describes a general etiquette and code of conduct that every team member should follow, as well as some concrete guidelines and activities we use in the team related to code reviews.
+The purpose of these guidelines is to formalise some basic requirements for pull requests and receiving/providing feedback to improve the quality of code reviews and teamwork in general.
+
+Note: Guidelines in this document should be considered to be of equal importance
+
+Note: This document is heavily inspired by [this blog post](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)
+
+Everyone
+--------
+
+* Be respectful to each other
+* Be humble. ("I'm not sure - let's look it up.")
+* Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+* Don't use sarcasm or trolling
+* If you feel annoyed at something/someone, don't make it affect your or others' work. Talk to a friend, go for a walk, meditate, switch to something else.
+* Talk synchronously, e.g. call or in person, if there are too many "I didn't understand" or "Alternative solution:" comments. Prefer not to use chat for that. Post a follow-up
+  comment summarizing the discussion.
+* Avoid selective ownership of code ("mine", "not mine", "yours"). Prefer referring to it as `our code`.
+* Use "we" instead of "you", unless asking for opinion of specific individual ("What do you think ...?", but "Here we are doing ... What if we ...?",)
```

Comment:
```suggestion
* Use "we" instead of "you", unless asking for someon's opinion ("What do you think ...?", but "Here we are doing ... What if we ...?",)
```